partition/roles/promtail/templates/promtail.yaml.j2

```
@@ -0,0 +1,74 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /var/log/promtail-positions.yaml
+
+clients:
+  - url: {{ promtail_loki_push_endpoint }}/loki/api/v1/push
```

```suggestion
  - url: {{ promtail_loki_push_endpoint }}
```
`/loki/api/v1/push` should be part of `promtail_loki_push_endpoint`.